rest/Cargo.toml

```diff
@@ -34,6 +34,10 @@ opentelemetry-jaeger = { version = "0.10", features = ["tokio"] }
 tracing-opentelemetry = "0.10.0"
 opentelemetry = "0.11.2"
 actix-web-opentelemetry = "0.9.0"
+paperclip = { version = "0.5.0", features = ["actix3-nightly"] }
```

Comment:
Hmmm, I note `paperclip` is explicitly noted to be WIP, and this feature is using nightly versions of actix.

I would recommend we don't use nightly, and limit how much we use this crate since it's explicitly warning us it's potentially unsuitable for release/production use.

Reply:
Sadly, there are not a lot of alternatives here :(
`paperclip` is only used to generate the spec, so its impact in the server code should be minimal and we should be able to test this. That said, we could feasibly use it for testing/devel when we want the spec/swagger and not use for production. This code is also wip will not be shipping yet, we can reevaluate what to do at that time.

Reply:
Gotcha! That makes more sense. :)